pkg/sshd/etc/profile

```diff
@@ -0,0 +1,13 @@
+export CHARSET=UTF-8
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+export PAGER=less
+export PS1='(sshd ns):\w\$ '
+export PS1='\h:\w\$ '
```

Comment:
They probably should @ijc. @justincormack raised that early on in getty's life. But we didn't want starting one to cause both to kick off. Someone might want just `getty`, another just `sshd`. And putting it in `init:` should not kick off `sshd` unless so desired.